src/components/vocabulary/Vocab.js

```diff
@@ -72,14 +129,27 @@ const vocabulary = {
     description: "URI property value",
     url: "http://sinopia.io/vocabulary/propertyType/uri",
   },
+  "resourceAttribute/suppressible": {
+    description:
+      "whether resource is suppressible (must have only one property which is a lookup or URI)",
+    url: "http://sinopia.io/vocabulary/resourceAttribute/suppressible",
+  },
   ResourcePropertyTemplate: {
     description: "Class for a resource property template",
     url: "http://sinopia.io/vocabulary/ResourcePropertyTemplate",
   },
   ResourceTemplate: {
-    description: "Class for a resource template",
+    description: "Class for a resource template ('template' or 'resource')",
     url: "http://sinopia.io/vocabulary/ResourceTemplate",
   },
+  Uri: {
+    description: "Class for a URI template",
+    url: "http://sinopia.io/vocabulary/Uri",
```

Comment:
it seems ... wrong ... to have a URI as a sinopia vocabulary ... isn't URI already defined?

Reply:
Dunno, but this vocab is primarily for internal use so not overwhelmingly concerned.